docs/userguides/dependencies.md

```diff
@@ -273,26 +273,33 @@ dependencies:
           - mocks/**/*      # Ignore all files in the 'mocks' directory
 ```
 
-### Solidity Remappings
+### Solidity Mapping
 
-A common use-case for dependencies involves the Solidity plugin.
-To use your dependencies in the `ape-solidity` plugin, configure `import_remappings` to refer to them:
+A common use-case for dependencies involves the Solidity plugin. By default, the `ape-solidity` plugin knows to look at installed dependencies for potential remapping-values and will use those when it notices you are importing them. For example, if you are using dependencies like:
```

Comment:
```suggestion
A common use-case for dependencies involves the Solidity plugin.
By default, the `ape-solidity` plugin knows to look at installed dependencies for potential remapping-values and will use those when it notices you are importing them.
For example, if you are using dependencies like:
```

Reply:
btw we typically like to do 1 sentence per line (markdown will turn consecutive sentences into one paragraph if there are no spaces between them)